### PR TITLE
Remove av logger from viam-server

### PR DESCRIPTION
**Pull request by randhid**

### CODE REVIEW

File: web/server/entrypoint.go [hunk 1]
```diff
@@ -22,7 +22,6 @@ import (
 	"go.viam.com/utils/perf"
 	"go.viam.com/utils/rpc"
 
-	vlogging "go.viam.com/rdk/components/camera/videosource/logging"
```
Comment (hexbabe) [inline web/server/entrypoint.go:25]: vlogging

File: components/camera/videosource/logging/logger.go [hunk 2]
```diff
@@ -1,304 +1,2 @@
-// Package logging is a thread-safe way to log video device information to a file. On startup, this package creates a
-// unique filename and uses that filename throughout the lifetime of the program to log information such as which video
-// devices are V4L2 compatible and the current operating system.
+// Package logging is to be removed
```
Comment (hexbabe) [inline components/camera/videosource/logging/logger.go:1]: Why not remove in this PR again?

**Review by hexbabe — APPROVED** [review]

File: components/camera/videosource/logging/logger.go (hunk 2 above)
Comment (randhid) [reply components/camera/videosource/logging/logger.go:1]: `TestShutdown` started doing weird things when I removed the packages themselves, going to do that in a separate commit when I figure out why.

**Review by seanavery — APPROVED** [review]
Thank you for removing jetsoncamera code 🎉 

Dido question on why not fully remove jetsoncamera package?

**Comment by randhid** [comment]
> Thank you for removing jetsoncamera code 🎉
> 
> Dido comment on fully removing jetsoncamera package.

Test was failing when I removed packages, so was looking at why

File: components/camera/server_test.go [hunk 3]
```diff
@@ -59,11 +60,11 @@ func TestServer(t *testing.T) {
 	test.That(t, png.Encode(&imgBuf, img), test.ShouldBeNil)
 	var imgBufJpeg bytes.Buffer
 
-	test.That(t, rimage.EncodeJPEG(&imgBufJpeg, img), test.ShouldBeNil)
```
Comment (randhid) [inline components/camera/server_test.go:62]: change back

**Comment by randhid** [comment]
Tasks:
- [ ] Darwin build + can view webcam stream
- [ ] Raspberry pi static build build + can view webcam stream
- [ ] x86 static build + can view webcam stream
- [ ] x86 build _ can view webcam stream

**Pull request merged**